src/docs/community/faq.mdx

```diff
@@ -1,36 +1,115 @@
+
+If you need to build a standalone desktop application, we recommend using a tool like [Electron](https://www.electronjs.org/). This will allow you to build the application using one of PX Blue's supported web technologies while still being able to package the application for installation onto a desktop.
+
+</FAQExpander>
+
+<FAQExpander question={`How can I contribute?`}>
+
+If you are interested in contributing to PX Blue, that's awesome! We welcome pull requests for features and bug fixes. We're also more than happy to have you report bugs and reach out to us with comments, suggestions, or other ideas. Please read our [Innersourcing](/community/innersourcing) guide for more details.
+
+</FAQExpander>
+
+
+
+## Designers
+
+<FAQExpander question={`Do you have any design assets for PX Blue?`}>
+
+Yes! We have standardized on Figma as our preferred tool for designing PX Blue applications. We have a [Component Stickersheet](https://www.figma.com/community/file/852558784352181868) as well as a [Design Pattern Stickersheet](https://www.figma.com/community/file/926189711301522231) that give you access to a wide array of design elements that you can use to kick start your projects. 
+
+</FAQExpander>
 
-We will be continuously updating the Design Patterns section of pxblue.github.io with new patterns and code samples as they are ready. We will prioritize patterns and examples based on interest expressed from development teams. If there is something you want to see, [Contact us](/community/contactus).
+<FAQExpander question={`Do you have design patterns for _____?`}>
+
+We will be continuously updating the [Design Patterns](/patterns) section with new patterns and code samples as they are available. We will prioritize patterns and examples based on interest expressed from development teams. If there is something you want to see, please contact us.
 
 If you have built something that you would like to share, check out our information on [innersourcing](/community/innersourcing) and [Contact Us](/community/contactus) with a demo.
+
+</FAQExpander>
+
+<FAQExpander question={`Wht do other PX Blue projects look like?`}>
+
+If you are interested in seeing some examples of past and present projects that have been designed using PX Blue, please get in touch with us. We would be more than happy to point you to some reference materials from other projects.
+
+</FAQExpander>
+
+<FAQExpander question={`How much creative freedom do I have as a designer?`}>
+
+PX Blue is not meant to be proscriptive in our recommendations. We want PX Blue products to feel like they are part of the same family, but we do not want them to all feel like the same application. There are some aspects of applications that need to be the same across products to drive this consistency, but other areas are fair game for you to develop your own style. You can learn more my reading our page on [Product Identity](/design/project-identity).
```

Comment:
```suggestion
PX Blue is not meant to be prescriptive in our recommendations. We want PX Blue products to feel like they are part of the same family, but we do not want them to all feel like the same application. There are some aspects of applications that need to be the same across products to drive this consistency, but other areas are fair game for you to develop your own style. You can learn more by reading our page on [Product Identity](/design/project-identity).
```

Reply:
The language here is a little confusing. Proscribe and Prescribe are kind of opposites. 

![image](https://user-images.githubusercontent.com/29152776/104369959-313c9100-54ec-11eb-80cb-259ad0c44a61.png)
![image](https://user-images.githubusercontent.com/29152776/104369983-38639f00-54ec-11eb-8b2e-dbb4b824f6f0.png)

Despite being opposites, I think a case could be made for using either one and having the same meaning lol. I will reword the sentence to avoid the word entirely.

Reply:
prescribe = you must do this
proscribe = you must not do this

Reply:
Want to unify the languages? This is on the proj identity page.

![image](https://user-images.githubusercontent.com/8997218/104385657-a1a1dd00-5501-11eb-9253-17baa2e512c3.png)